lstchain/tools/lstchain_create_time_calibration_file.py

```diff
@@ -0,0 +1,108 @@
+"""
+Create drs4 time correction coefficients.
+"""
+import glob
+import numpy as np
+
+from ctapipe.core import Provenance, traits
+from ctapipe.core import Tool
+from ctapipe_io_lst import LSTEventSource
+from lstchain.calib.camera.r0 import LSTR0Corrections
+from lstchain.calib.camera.time_correction_calculate import TimeCorrectionCalculate
+
+
+class TimeCalibrationHDF5Writer(Tool):
+
+    name = "TimeCalibrationHDF5Writer"
+    description = "Generate a HDF5 file with time calibration coefficients"
+
+    input_file = traits.Unicode(
+        help="Path to the input file containing events (wildcards allowed)",
+    ).tag(config=True)
+
+    output_file = traits.Unicode(
+        help="Path to the time calibration file",
+    ).tag(config=True)
+
+    pedestal_file = traits.Unicode(
+        help="Path to drs4 pedestal file",
+    ).tag(config=True)
+
+    max_events = traits.Int(
+        help="Maximum numbers of events to read. Default = 20000",
+        default_value=2000
```

Comment:
Hi @Bultako,

could you please set the default value of max_events to 20000?  (this is what we use now in the online script, 2000 is not enough)

Reply:
oups, well spotted !!
I've added the missing 0